keps/prod-readiness/sig-auth/5284.yaml

```
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 5284
+alpha:
+  approver: "enj"
```

```suggestion
  approver: "deads2k"
```

He added himself to this and he is already assigned in our board.